Index: app/src/main/java/com/example/scheduler2/courses_adapter_data.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/scheduler2/courses_adapter_data.java	(revision 07d1a36439fb1e68357fd99245a51cce5ac5f3eb)
+++ app/src/main/java/com/example/scheduler2/courses_adapter_data.java	(revision 5523773b9181e08e80cfef6847d7be4efc2a4778)
@@ -4,6 +4,7 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.AdapterView;
 import android.widget.TextView;
 
 import androidx.recyclerview.widget.RecyclerView;
@@ -14,6 +15,7 @@
 
     private Context context;
    private ArrayList<Course_display> list;
+   //private OnItemClickListener mListener;;
     public courses_adapter_data(Context context, ArrayList<Course_display> list) {
         this.context = context;
         this.list = list;
@@ -43,7 +45,7 @@
         return list.size();
     }
 
-    public class ViewHolder extends RecyclerView.ViewHolder {
+    public class ViewHolder extends RecyclerView.ViewHolder{
         private TextView course_name, course_day, course_room_number, course_professor_name;
         public ViewHolder(View itemView) {
             super(itemView);
@@ -51,9 +53,11 @@
             course_day = itemView.findViewById(R.id.course_day);
             course_room_number = itemView.findViewById(R.id.course_room_number);
             course_professor_name = itemView.findViewById(R.id.course_professor_name);
+            //itemView.setOnClickListener(this);
+
+        }
 
 
-        }
     }
 }
 
